fe/fe-core/src/main/jflex/sql_scanner.flex

```diff
@@ -270,7 +270,7 @@ import org.apache.doris.qe.SqlModeHelper;
         keywordMap.put("isolation", new Integer(SqlParserSymbols.KW_ISOLATION));
         keywordMap.put("job", new Integer(SqlParserSymbols.KW_JOB));
         keywordMap.put("join", new Integer(SqlParserSymbols.KW_JOIN));
-        keywordMap.put("json", new Integer(SqlParserSymbols.KW_JSON));
+        keywordMap.put("jsonb", new Integer(SqlParserSymbols.KW_JSONB));
```

Comment:
why changing `json` to `jsonb`?
I think `jsonb` is an internal implementation, so for user interface, it is better be "json"?